<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GatherViewer</class>
 <widget class="QMainWindow" name="GatherViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GatherViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icons/images/logo_fs_64x64.png</normaloff>:/icons/images/logo_fs_64x64.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="GatherView" name="gatherView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>980</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="exitAct"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>S&amp;cale</string>
    </property>
    <addaction name="actionSet_Scale"/>
    <addaction name="separator"/>
    <addaction name="zoomInAct"/>
    <addaction name="zoomOutAct"/>
    <addaction name="separator"/>
    <addaction name="zoomFitWindowAct"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionTrace_Scaling"/>
    <addaction name="separator"/>
    <addaction name="action_Trace_Options"/>
    <addaction name="actionVolume_Options"/>
    <addaction name="action_Horizon_Display_Options"/>
    <addaction name="action_Point_Display_Options"/>
    <addaction name="actionPick_Display_Options"/>
    <addaction name="actionDensity_Color_Table"/>
    <addaction name="actionVolume_Color_Table"/>
    <addaction name="actionGather_Histogram"/>
    <addaction name="actionVolume_Histogram"/>
    <addaction name="actionVolume_Display_Range"/>
   </widget>
   <widget class="QMenu" name="menu_Broadcast">
    <property name="title">
     <string>&amp;Broadcast</string>
    </property>
    <addaction name="action_Receive_CDPs"/>
    <addaction name="action_Dispatch_CDPs"/>
    <addaction name="separator"/>
    <addaction name="actionShare_Current_Position"/>
   </widget>
   <widget class="QMenu" name="menu_Window">
    <property name="title">
     <string>&amp;Window</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Picking">
    <property name="title">
     <string>Pic&amp;king</string>
    </property>
    <widget class="QMenu" name="menuPick_Mode">
     <property name="title">
      <string>&amp;Pick Mode</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuPick_Type">
     <property name="title">
      <string>P&amp;ick Type</string>
     </property>
    </widget>
    <widget class="QMenu" name="menuFill_Method">
     <property name="title">
      <string>&amp;Fill Method</string>
     </property>
    </widget>
    <addaction name="action_New_Picks"/>
    <addaction name="action_Load_Picks"/>
    <addaction name="action_Save_Picks"/>
    <addaction name="action_Close_Picks"/>
    <addaction name="separator"/>
    <addaction name="menuPick_Type"/>
    <addaction name="menuFill_Method"/>
    <addaction name="menuPick_Mode"/>
    <addaction name="actionConservative"/>
   </widget>
   <widget class="QMenu" name="menu_Items">
    <property name="title">
     <string>&amp;Items</string>
    </property>
    <addaction name="actionSeismic_Dataset_Info"/>
    <addaction name="separator"/>
    <addaction name="actionSetup_Horizons"/>
    <addaction name="actionSetup_Volume"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menu_Items"/>
   <addaction name="menuDisplay"/>
   <addaction name="menu_Picking"/>
   <addaction name="menu_Window"/>
   <addaction name="menu_Broadcast"/>
  </widget>
  <widget class="QToolBar" name="mouseToolBar">
   <property name="windowTitle">
    <string>Mouse Mode Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="zoomToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Zoom Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="zoomInAct"/>
   <addaction name="zoomOutAct"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>General Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionTrace_Scaling"/>
   <addaction name="actionDensity_Color_Table"/>
   <addaction name="separator"/>
   <addaction name="action_Trace_Options"/>
   <addaction name="actionVolume_Options"/>
   <addaction name="separator"/>
   <addaction name="action_Horizon_Display_Options"/>
   <addaction name="action_Point_Display_Options"/>
   <addaction name="actionPick_Display_Options"/>
   <addaction name="actionSeismic_Dataset_Info"/>
   <addaction name="actionGather_Histogram"/>
   <addaction name="actionVolume_Histogram"/>
   <addaction name="separator"/>
   <addaction name="action_Receive_CDPs"/>
   <addaction name="action_Dispatch_CDPs"/>
   <addaction name="actionShare_Current_Position"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="pickToolBar">
   <property name="windowTitle">
    <string>Pick Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="navigationToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>Navigation Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="gainToolBar">
   <property name="windowTitle">
    <string>Gain Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="exitAct">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="zoomInAct">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Zoom-Plus-icon.png</normaloff>:/icons/Zoom-Plus-icon.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom In (25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="zoomOutAct">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/Zoom-Minus-icon.png</normaloff>:/icons/Zoom-Minus-icon.png</iconset>
   </property>
   <property name="text">
    <string>Zoom &amp;Out (25%)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="zoomFitWindowAct">
   <property name="text">
    <string>Zoom &amp;Fit Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="openGridAct">
   <property name="text">
    <string>&amp;Horizon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="closeGridAct">
   <property name="text">
    <string>&amp;Horizon</string>
   </property>
  </action>
  <action name="action_Receive_CDPs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/receive-icon-32x32.png</normaloff>:/icons/images/receive-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Receive CDPs</string>
   </property>
  </action>
  <action name="action_Dispatch_CDPs">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/send-icon-32x32.png</normaloff>:/icons/images/send-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Dispatch CDPs</string>
   </property>
  </action>
  <action name="actionOpenVolume">
   <property name="text">
    <string>&amp;Volume</string>
   </property>
  </action>
  <action name="actionCloseVolume">
   <property name="text">
    <string>&amp;Volume</string>
   </property>
  </action>
  <action name="actionVolume_Options">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/volume-display-options-icon-32x32.png</normaloff>:/icons/images/volume-display-options-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Volume Display Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Trace_Options">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/trace-display-options-icon-32x32.png</normaloff>:/icons/images/trace-display-options-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Trace Display Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionTrace_Scaling">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/trace-scaling-icon-32x32.png</normaloff>:/icons/images/trace-scaling-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Trace &amp;Scaling</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Point_Display_Options">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/point-display-options-icon-32x32.png</normaloff>:/icons/images/point-display-options-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Point Display Options</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionShare_Current_Position">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/current-point-icon-32x32.png</normaloff>:/icons/images/current-point-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Share Current Position</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionDensity_Color_Table">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/colortable-icon-32x32.png</normaloff>:/icons/images/colortable-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Density &amp;Color Table</string>
   </property>
  </action>
  <action name="actionSet_Scale">
   <property name="text">
    <string>Set &amp;Scale</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionVolume_Color_Table">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/volume-colortable-icon-32x32.png</normaloff>:/icons/images/volume-colortable-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>V&amp;olume Color Table</string>
   </property>
  </action>
  <action name="action_Load_Picks">
   <property name="text">
    <string>&amp;Load Picks</string>
   </property>
  </action>
  <action name="action_Save_Picks">
   <property name="text">
    <string>&amp;Save Picks</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="action_Close_Picks">
   <property name="text">
    <string>&amp;Close Picks</string>
   </property>
  </action>
  <action name="actionSingle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Single</string>
   </property>
  </action>
  <action name="actionMulti">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Multi</string>
   </property>
  </action>
  <action name="actionMinimum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/pick-minimum-32x32.png</normaloff>:/icons/images/pick-minimum-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Minimum</string>
   </property>
  </action>
  <action name="actionMaximum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/pick-maximum-32x32.png</normaloff>:/icons/images/pick-maximum-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Ma&amp;ximum</string>
   </property>
  </action>
  <action name="action_New_Picks">
   <property name="text">
    <string>&amp;New Picks</string>
   </property>
  </action>
  <action name="actionGather_Histogram">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/histogram-gather-icon-32x32.png</normaloff>:/icons/images/histogram-gather-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Gather Histogram</string>
   </property>
  </action>
  <action name="actionVolume_Histogram">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/histogram-volume-icon-32x32.png</normaloff>:/icons/images/histogram-volume-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Vo&amp;lume Histogram</string>
   </property>
  </action>
  <action name="actionConservative">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Retain Picks</string>
   </property>
  </action>
  <action name="actionFill_Strategy">
   <property name="text">
    <string>Fill Strategy</string>
   </property>
  </action>
  <action name="actionFill_Strategy_2">
   <property name="text">
    <string>Fill Strategy</string>
   </property>
  </action>
  <action name="actionGg">
   <property name="text">
    <string>gg</string>
   </property>
  </action>
  <action name="actionVolume_Display_Range">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/images/select-range-icon-32x32.png</normaloff>:/icons/images/select-range-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Volume &amp;Display Range</string>
   </property>
  </action>
  <action name="actionSet_Correlation_Window">
   <property name="text">
    <string>S&amp;et Correlation Window</string>
   </property>
  </action>
  <action name="actionSet_Search_Window">
   <property name="text">
    <string>Set Search &amp;Window</string>
   </property>
  </action>
  <action name="actionPick_Display_Options">
   <property name="icon">
    <iconset>
     <normaloff>:/images/pick-display-options-icon-32x32.png</normaloff>:/images/pick-display-options-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>P&amp;ick Display Options</string>
   </property>
  </action>
  <action name="actionSeismic_Dataset_Info">
   <property name="icon">
    <iconset>
     <normaloff>:/images/info-icon-32x32.png</normaloff>:/images/info-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>Seismic Dataset &amp;Info</string>
   </property>
  </action>
  <action name="actionSetup_Horizons">
   <property name="text">
    <string>Setup &amp;Horizons</string>
   </property>
  </action>
  <action name="actionSetup_Volume">
   <property name="text">
    <string>Setup &amp;Volume</string>
   </property>
  </action>
  <action name="action_Horizon_Display_Options">
   <property name="icon">
    <iconset>
     <normaloff>:/images/horizon-display-options-icon-32x32.png</normaloff>:/images/horizon-display-options-icon-32x32.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Horizon Display Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GatherView</class>
   <extends>QWidget</extends>
   <header location="global">gatherview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>exitAct</sender>
   <signal>triggered()</signal>
   <receiver>GatherViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
